server/src/utils.ts

```diff
@@ -16,7 +17,7 @@ type RawPagination = {
 }
 
 type JSONResponse = {
-    items: Object;
+    items: {};
     pagination: RawPagination;
 };
 
```

Comment:
I guess `rawRequest` and `paginatedRequest` can be removed from here as they are in `cloudify/rest.ts` already, right?

Reply:
Yes. Utils will be deleted.